lxml.h

```diff
@@ -318,7 +373,9 @@ int XMLDocument_load(XMLDocument* doc, const char* path)
                     return FALSE;
                 }
 
-                curr_node->inner_text = strdup(lex);
+                if(NULL == curr_node->inner_text)
```

Comment:
This was sort of weird and was leaking memory. I wasn't quite sure what it was doing and added a `FIXME` to say `realloc` + `strcat` but this only resulted in odd `inner_text` that was fragmented. There is a question of whether this logic is correct, for the moment I've just removed the memory leak.